pnpm-lock.yaml

```diff
@@ -17914,6 +17918,15 @@ packages:
       '@fluidframework/gitresources': 4.0.0
       '@fluidframework/protocol-definitions': 3.2.0
       events: 3.3.0
+    dev: true
+
+  /@fluidframework/protocol-base/5.0.0-262463:
+    resolution: {integrity: sha512-DxOqSucs2jtVddp5jVc5EOzwCCi35WuA8SpvFT94Z/ESrb8JpHhK55OGgK4AVsz7QavuWlg0/B9FQjfHclhOhw==}
+    dependencies:
+      '@fluidframework/common-utils': 3.1.0
+      '@fluidframework/gitresources': 5.0.0-262463
+      '@fluidframework/protocol-definitions': 3.2.0
+      events_pkg: /events/3.3.0
```

Comment:
assuming a prerelease dependency is okay, you will need to figure out the duplications that are now appearing. What is using older 4.x and is that okay?